src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -205,14 +208,17 @@ export const useSwapper = () => {
     [swapperManager, sellTradeAsset],
   )
 
-  const getDefaultPair = useCallback((connectedChainId: ChainId | undefined) => {
-    switch (connectedChainId) {
+  const getDefaultPair = useCallback((buyAssetChainId: ChainId | undefined) => {
+    switch (buyAssetChainId) {
       case KnownChainIds.AvalancheMainnet:
         return [avalancheAssetId, 'eip155:43114/erc20:0x49d5c2bdffac6ce2bfdb6640f4f80f226bc10bab']
+      case KnownChainIds.CosmosMainnet:
+        return [cosmosAssetId, osmosisAssetId]
```

Comment:
unfortunately, we can't merge this as is, until osmo flag is enabled, as the osmo asset id isn't present with the flag off.

Reply:
Good catch. 

Put behind feature flag in https://github.com/shapeshift/web/pull/2370/commits/5fd485551d4eecd9544ee4a594347a48d53dc984.